src/app/core/data/signposting-data.service.ts

```diff
@@ -25,7 +25,8 @@ export class SignpostingDataService {
    * @param uuid
    */
   getLinks(uuid: string): Observable<SignpostingLink[]> {
-    const baseUrl = this.halService.getRootHref().replace('/api', '');
+    const regex = /\/api$/gm;
+    const baseUrl = this.halService.getRootHref().replace(regex, '');
```

Comment:
I'm sure this works, but why do this replace at all? I'd [inject appConfig](https://github.com/DSpace/dspace-angular/blob/9a433b50ff8f9e610fe0ba242d035b3f5c8a2f28/src/app/core/reload/reload.guard.ts#L14) and use `this.appconfig.rest.baseUrl`

Reply:
@artlowel 

good hint, thanks. fixed